lib/skate_web/channels/user_socket.ex

```diff
@@ -22,9 +23,26 @@ defmodule SkateWeb.UserSocket do
   # See `Phoenix.Token` documentation for examples in
   # performing token verification on connect.
   def connect(%{"token" => token}, socket, _connect_info) do
+    user_id =
```

Comment:
suggestion(non-blocking): What do you think about naming this variable differently so it's obvious that this is not safe to use for anything but logging?
Maybe `unreliable_user_id`, or `unconfirmed_user_id`. I was thinking `unvalidated_user_id` which is maybe better, but I'm not sure `unvalidated` is a proper word?

Reply:
How about `unverified`? I think that matches up with Guardian terminology.

Reply:
works for me 👍🏻

Reply:
cd5e4940